AutomationGui.sc

```diff
@@ -219,20 +219,25 @@ AutomationGui {
             bsnap.top = brec.top + brec.height;
             bsnap.height = min(bsnap.width, 0.1 * bounds.height);
         };
-        snapshotBtn = GUI.button.new( win, bsnap );
-        snapshotBtn.states = [["[ô]", Color.black, Color.grey],
-                              ["[ô]", Color.white, Color.red]];
-
-        snapshotBtn.action = {
-            automation.snapshot;
-            {
-                snapshotBtn.value = 0;
-            }.defer(0.2);
-        };
+	if (automation.showSnapshot){
```

Comment:
this is using a tab to indent while the rest of the file uses only spaces. Same below.

In this github view of the patch, the indenting looks one off, that's because of the tab. The file looks correct, but because the diff has a '+' in the first column, the eight spaces become nine in the diff, while the tab 'swallows' the '+' and remains eight.